api/jobs.yaml

```diff
@@ -882,10 +882,6 @@ definitions:
         type: array
         items:
           $ref: '#/definitions/IndividualAttempt'
-      executionEvents:
```

Comment:
This looks like it is still a field in the response (ie and only the fields in it have changed)?

Reply:
`executionEvents` are now only returned to the UI within the `TaskMetadata`, not when the UI requests attempt-specific data.  Otherwise, those events would not be available to the timing diagram component to render properly.